AardvarkLoggingUI.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name     = 'AardvarkLoggingUI'
+  s.version  = '1.0.0'
+  s.license  = 'Apache License, Version 2.0'
+  s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
+  s.homepage = 'https://github.com/square/Aardvark'
+  s.authors  = 'Square'
+  s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => "AardvarkLoggingUI/#{ s.version.to_s }" }
+
+  s.swift_version = '4.0'
+  s.ios.deployment_target = '8.0'
+
+  s.source_files = 'Sources/AardvarkLoggingUI/**/*.{h,m,swift}'
```

Comment:
Lazy q: are these files available via `Aardvark` anymore? If not, I think we need to do a major version bump on `Aardvark`...

Reply:
Nope. And correct, this will be a major version bump for `Aardvark`. Going to update all of the podspecs in a separate PR, to try to break this refactor into small, logical changes.